pyproject.toml

```diff
@@ -41,7 +41,7 @@ crashtest = "^0.4.1"
 dulwich = "^0.21.2"
 importlib-metadata = { version = ">=4.4", python = "<3.10" }
 installer = "^0.7.0"
-jsonschema = "^4.10.0"
+jsonschema = ">=4.10.0,<4.18.0"
```

Comment:
Can you add a comment, why we are restricting it?

Reply:
Done